### PR TITLE
Bluetooth UUID unit tests

### PR DESCRIPTION
**Pull request by fokinv**
This is a unit test for the BluetoothUUID interface's functions.

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors

### CODE REVIEW

File: tests/unit/script/bluetooth_uuid.rs [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    let invalid_character = StringOrUnsignedLong::String(DOMString::from(INVALID_CHARACTER_UUID));
+    assert_eq!(format!("{:?}", BluetoothUUID::service(invalid_character.clone()).unwrap_err()), SYNTAX_ERROR);
+    assert_eq!(format!("{:?}", BluetoothUUID::characteristic(invalid_character.clone()).unwrap_err()), SYNTAX_ERROR);
+    assert_eq!(format!("{:?}", BluetoothUUID::descriptor(invalid_character).unwrap_err()), SYNTAX_ERROR);
+}
+
+#[test]
+fn valid_uuid_from_name() {
+    let alert_notification = StringOrUnsignedLong::String(DOMString::from(ALERT_NOTIFICATION));
+    let uuid1 = DOMString::from(ALERT_NOTIFICATION_UUID);
```
Comment (zakorgy) [inline tests/unit/script/bluetooth_uuid.rs]: Maybe rename `uuid1`, `uuid2`, `uuid3`, to `alert_notification_uuid` etc.

File: tests/unit/script/bluetooth_uuid.rs [hunk 2]
```diff
@@ -0,0 +1,131 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use script::dom::bindings::codegen::UnionTypes::StringOrUnsignedLong;
+use script::dom::bindings::str::DOMString;
+use script::dom::bluetoothuuid::BluetoothUUID;
+
+const ADEADBEEF_ALIAS: &'static StringOrUnsignedLong = &StringOrUnsignedLong::UnsignedLong(0xADEADBEEF);
+const AEROBIC_HEART_RATE: &'static str = "aerobic_heart_rate_lower_limit";
```
Comment (zakorgy) [inline tests/unit/script/bluetooth_uuid.rs:10]: This string is only used to create `DOMString`s can't we just create the DOMString here?

File: tests/unit/script/bluetooth_uuid.rs [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+const AEROBIC_HEART_RATE_UUID: &'static str = "00002a7e-0000-1000-8000-00805f9b34fb";
```
Comment (zakorgy) [inline tests/unit/script/bluetooth_uuid.rs:11]: Same as above

**Review by zakorgy — COMMENTED** [review]
Can we create `&'static DOMString`s instead of `&'static str`.

File: tests/unit/script/bluetooth_uuid.rs (hunk 2 above)
Comment (fokinv) [reply tests/unit/script/bluetooth_uuid.rs:10]: Sadly, we can't.

**Pull request closed without merging**